defaults/main.yml

```diff
@@ -62,6 +62,12 @@ he_vm_mac_addr: null
 he_remove_appliance_rpm: true
 he_pki_renew_on_restore: false
 
+# If you decide to change these variables,
+# make sure they have the subnet mask
+he_local_vm_net_addr: "192.168.222.1"
+he_local_vm_dhcp_start: "192.168.222.2"
+he_local_vm_dhcp_end: "192.168.222.254"
+
```

Comment:
For IPv6 we are just using he_ipv6_subnet_prefix.
I think we should do something analogue here:
he_ipv4_subnet_prefix: "192.168.222"
and then compute everything else starting from there and assuming a /24 subnet

Reply:
I thought about that, thought to give the user more flexibility playing with the ranges, but I'm guessing that's not that important.
I will change to he_ipv4_subnet_prefix